include/mega/types.h

```diff
@@ -227,6 +227,13 @@ typedef enum { TYPE_UNKNOWN = -1, FILENODE = 0, FOLDERNODE, ROOTNODE, INCOMINGNO
 const int FILENODEKEYLENGTH = 32;
 const int FOLDERNODEKEYLENGTH = 16;
 
+
+//handle lengths
+static const int PUBLICHANDLE = 6;
+
+//handle lengths
+static const int USERHANDLE = 8;
+
```

Comment:
It would be probably better to use the already existing `MegaClient::USERHANDLE` and `MegaClient::NODEHANDLE` or, if `PUBLICHANDLE` is not related to nodes, add the new value there. Anyway, it seems that these definitions are not being used.